### DISCUSSION TITLE
Remapでのmouse wh の設定について

### DISCUSSION THREAD

**Discussion by kodaishida** [Yowkees/keyball#711] [Q&A]
最新ファームウェアに更新したkeyball61を使っています。
ペンタブを併用しておりボールなしでもホイール操作がしたいのですがキーマップでmouse whを設定しても反応しない状態です。自力での書き換えなど必要な仕様でしょうか？
以前のDiscussionでも同様の質問があったことは承知の上ですが数年前のものだったため改めて質問させていただきます。

**Comment by kodaishida** [comment]
こちら参考にMOUSEKEY_ENABLE = yesと設定したところマウスホイールの操作が可能になりました。失礼いたしました。

https://blog.kiamotomonic.net/mumbojumbo/keyball-firmware/

**Comment by Yowkees** [comment]
ご自身で解決していただき、誠にありがとうございます。
ホイール操作をキーで行った経験がなかったので参考になります。
